icons/alarm-clock.svg

```diff
@@ -9,10 +9,10 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="13" r="8" />
-  <path d="M12 9v4l2 2" />
-  <path d="M5 3 2 6" />
-  <path d="m22 6-3-3" />
-  <path d="m6 19-2 2" />
-  <path d="m18 19 2 2" />
+  <circle cx="12" cy="13" r="8"/>
+  <path d="M12 9v4l2 2"/>
+  <path d="M5 3 2 6"/>
+  <path d="m22 6-3-3"/>
+  <path d="M6.38 18.7 4 21"/>
+  <path d="M17.64 18.67 20 21"/>
```

Comment:
(In the future feel free to post the same fix of multiple icons in a single PR, as this one's the same as in #1077 #1083 #1082 #1078 ).

```suggestion
  <circle cx="12" cy="13" r="8" />
  <path d="M12 9v4l2 2" />
  <path d="M5 3 2 6" />
  <path d="m22 6-3-3" />
  <path d="M6.38 18.7 4 21" />
  <path d="M17.64 18.67 20 21" />
```